pkg/tanka/static.go

```diff
@@ -18,6 +18,12 @@ func (s StaticLoader) Load(path string, opts LoaderOpts) (*v1alpha1.Environment,
 		return nil, err
 	}
 
+	envCode, err := specToExtCode(*config)
+	if err != nil {
+		return nil, err
+	}
+	opts.ExtCode.Set(environmentExtCode, envCode)
```

Comment:
This will cause problems in parallel, I spend quite a few hours debugging this in the past. 

This is the solution I came up with:
https://github.com/grafana/tanka/pull/450/commits/a583ca84d6fa6d889d309790fcf253f4010195b5

Reply:
Tell me more, what happens?

Reply:
It seems to be hard to reproduce. This is the stacktrace: https://gist.github.com/Duologic/c3c7843cff15ced8cac0e17958f6c4a7

Reply:
@Duologic I fixed that race condition in 79d2de8f353f0bb3da8beb5f983b17f084d6134d and left some notes on the cause there

Reply:
Awesome. What if we clone each time we call Set() ?

Reply:
Out of curiosity, why is JsonnetOpts being updated inside a goroutine? Isn't it a static representation of command line args that should be considered read only by goroutines?

Reply:
`JsonnetOpts` also holds extVars, and we use that for providing `tk.env`. I do agree to your point though, that `somethingOpts` should generally be static, so we might work on our design here.

This would exceed this PR by far however, so we should tackle that in a separate one